### PR TITLE
Megan's PR

### PR DESCRIPTION
**Pull request by vanwars**
Megan,

Something must have happened to your local config, because for whatever reason, your Megan-Phase1-May5 branch is flagged as an "entirely different commit history from master." I made a few changes so that thing can merge. Please pull when you get a chance.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+app.post('', routes.postForm)
+
+app.get('/', (request, response) => {
+    return response.status(200).send({
+        'OK': 'Success'
+    })
+})
+
+/*
+// Note: We will worry about comments next week:
```
Comment (vanwars) [inline server/index.js:32]: Delete these comments: they're not necessary for this PR.

File: server/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const database = []
+
+module.exports = {
```
Comment (vanwars) [inline server/routes/index.js:3]: Nice!

**Comment by vanwars** [comment]
@ayyrickay @SomeSN @Noi-Git @Saranrat13 Can two of you please approve this PR. Looks good to me! Resolves Issue #15.

**Review by Miss-Megan — APPROVED** [review]
Not sure what binary file not shown means. is that important?

**Review by Miss-Megan — APPROVED** [review]

**Review by Miss-Megan — APPROVED** [review]

**Comment by vanwars** [comment]
Re: "binary file not shown." Not important. That message means that the .DS_Store file that you committed (which is a Mac System file) is not a text file...it's compiled. In general, you want to exclude system files (so I went ahead and added it to the .gitignore). OK for now, but you can also remove it.

**Review by Miss-Megan — APPROVED** [review]
@vanwars I had some trouble adding my files (and accidentally uploaded a bunch of extra files), so i deleted my github history before adding/deleting another branch. thank you so much for fixing all of this. especially on Saturday night :)

**Review by Noi-Git — APPROVED** [review]

**Review by Miss-Megan — APPROVED** [review]

**Comment by Miss-Megan** [comment]
@vanwars should i add the above files in the client folder in the terminal to resolve the conflicts so they can merge into master?

**Comment by vanwars** [comment]
I went ahead and resolved these conflicts. That said, Noi's PR should not
have been approved because her main.js file does not post to
localhost:3000. She's really close, but needs to make a few more changes.

**Comment by Noi-Git** [comment]
Ok will try to get it done before lab on Tuesday. Thanks Sarah.

Noi

**Pull request merged**